Libraries/VvVtkWidgets/vvVideoQueryDialogPrivate.cxx

```diff
@@ -293,7 +293,7 @@ void vvVideoQueryDialogPrivate::setTimeConstraint(
 void vvVideoQueryDialogPrivate::updateKeyframeItem(
   QTreeWidgetItem* item, vtkIdType id, const vgRegionKeyframe& keyframe)
 {
-  static const auto name = QStringLiteral("%1,%2 %3\u00d7%4");
+  static const auto name = QStringLiteral(u"%1,%2 %3\u00d7%4");
```

Comment:
This change had no obvious effect on the (VS2017) build.

Reply:
The multiplication symbol (×) is also in Latin-1.